src/features/profile/ProfileSlice.ts

```diff
@@ -0,0 +1,256 @@
+    { value: 'es-cl', label: 'Spanish (Chile)' },
+    { value: 'es-co', label: 'Spanish (Colombia)' },
+    { value: 'es-cr', label: 'Spanish (Costa Rica)' },
+    { value: 'es-do', label: 'Spanish (Dominican Republic)' },
+    { value: 'es-ec', label: 'Spanish (Ecuador)' },
+    { value: 'es-sv', label: 'Spanish (El Salvador)' },
+    { value: 'es-gt', label: 'Spanish (Guatemala)' },
+    { value: 'es-hn', label: 'Spanish (Honduras)' },
+    { value: 'es-mx', label: 'Spanish (Mexico)' },
+    { value: 'es-ni', label: 'Spanish (Nicaragua)' },
+    { value: 'es-pa', label: 'Spanish (Panama)' },
+    { value: 'es-py', label: 'Spanish (Paraguay)' },
+    { value: 'es-pe', label: 'Spanish (Peru)' },
+    { value: 'es-pr', label: 'Spanish (Puerto Rico)' },
+    { value: 'es-es', label: 'Spanish (Spain)' },
+    { value: 'es-uy', label: 'Spanish (Uruguay)' },
+    { value: 'es-ve', label: 'Spanish (Venezuela)' },
+    { value: 'sx', label: 'Sutu' },
+    { value: 'sw', label: 'Swahili' },
+    { value: 'sv', label: 'Swedish' },
+    { value: 'sv-fi', label: 'Swedish (Finland)' },
+    { value: 'sv-sv', label: 'Swedish (Sweden)' },
+    { value: 'ta', label: 'Tamil' },
+    { value: 'tt', label: 'Tatar' },
+    { value: 'te', label: 'Teluga' },
+    { value: 'th', label: 'Thai' },
+    { value: 'tig', label: 'Tigre' },
+    { value: 'ts', label: 'Tsonga' },
+    { value: 'tn', label: 'Tswana' },
+    { value: 'tr', label: 'Turkish' },
+    { value: 'tk', label: 'Turkmen' },
+    { value: 'uk', label: 'Ukrainian' },
+    { value: 'hsb', label: 'Upper Sorbian' },
+    { value: 'ur', label: 'Urdu' },
+    { value: 've', label: 'Venda' },
+    { value: 'vi', label: 'Vietnamese' },
+    { value: 'vo', label: 'Volapuk' },
+    { value: 'wa', label: 'Walloon' },
+    { value: 'cy', label: 'Welsh' },
+    { value: 'xh', label: 'Xhosa' },
+    { value: 'ji', label: 'Yiddish' },
+    { value: 'zu', label: 'Zulu' },
+  ],
+  loading: false,
+}
+
+export const pastelProfileSlice = createSlice({
+  name: 'pastelProfile',
+  initialState,
+  reducers: {
+    getLocations(state: TProfileState, action: PayloadAction<TSelect[]>) {
+      state.locations = action.payload
+      state.loading = true
+    },
+    getLocationsFailure(state: TProfileState) {
+      state.loading = false
+    },
+  },
+})
+
+const { getLocations, getLocationsFailure } = pastelProfileSlice.actions
+
+export const pastelProfileReducer = pastelProfileSlice.reducer
+
+export function fetchLocations(suggested: string): AppThunk {
+  return async dispatch => {
+    try {
+      console.log(suggested)
+      const res = await axios.get(
+        `https://nominatim.openstreetmap.org/search?q=${suggested}&format=json`,
+      )
+      const temp = []
+      for (const location of res.data) {
+        temp.push({
+          label: location.display_name,
+          value: location.display_name,
+        })
+      }
+      dispatch(getLocations(temp))
+    } catch (err) {
```

Comment:
Log the error in this way `log.error('Failed to fetch locations', error.message)` using 'electron-log' package

Reply:
fixed